3.6/session/main.md

```diff
@@ -82,6 +82,19 @@ echo $f3->get('SESSION.test');
 It will automatically create the required table, if the `session` table does not exist. The table name can be controlled by the 2nd `$table` parameter of the constructor. A 3rd parameter is used to `$force` the table creation, if it's not existing.
 
 
+The `$onsuspect` argument is there to override the default handling of suspicious sessions:
```

Comment:
The added text is a copy of the description in the `Cache` section above. The description is valid for all session backends, but only `Cache` and `SQL` backends are considered here. The "Events" section, at the end of the document, contains another description of the `onsuspect` feature.